src/xdppcw/inc/xdppcw.h

```diff
@@ -32,6 +32,7 @@ typedef struct _XDP_PCW_LWF_RX_QUEUE {
     UINT64 ForwardingNbsRequested;
     UINT64 ForwardingNbsSent;
     UINT64 LoopbackNblsSkipped;
+    UINT64 FramesInvalidChecksumOffload;
```

Comment:
Does this ever get set? If this is copied from the TX PR, I don't think it's applicable to RX since the user mode program doesn't specify the checksum for RX, unlike TX. Instead, the NIC should always give us a checksum.

Reply:
Sure, will address this.